apps/dashboard/src/components/primitives/button.tsx

```diff
@@ -1,36 +1,34 @@
-import * as React from "react";
-import { Slot } from "@radix-ui/react-slot";
-import { cva, type VariantProps } from "class-variance-authority";
-import { cn } from "@/utils/ui";
+import * as React from 'react';
```

Comment:
@desiprisg I was under the impression that the `primitives` folder is where we download the ShadUI code. 

What's the recommended framework for applying modifications to the ShadUI code? Do it in there or in a different place?

Reply:
You can directly edit the primitives and variants as needed. That's the difference between shadcn/ui and conventional ui libraries.

Reply:
https://ui.shadcn.com/docs#:~:text=Pick%20the%20components%20you%20need.%20Copy%20and%20paste%20the%20code%20into%20your%20project%20and%20customize%20to%20your%20needs.%20The%20code%20is%20yours.